### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by sneakernuts** [dani-garcia/vaultwarden#1854] [Q&A] [answered]
### Subject of the issue

After updating to the latest version, browser extension requires master password at every login
### Deployment environment

* vaultwarden version: 1.22.1

* Install method: Docker Image

* Clients used:  Browser extension

* Reverse proxy and version:  Traefik 1.7

* MySQL/MariaDB or PostgreSQL version:  MySQL

* Other relevant details: 

### Steps to reproduce

Update to latest docker image, browser extension prompts to enter in master password when looking up logins. 

### Expected behaviour

Extension should not have changed. Master password should have been governed by only the timeout setting

### Actual behaviour

Keeps getting prompted when using the extension. Sync did not happen automatically.

### Troubleshooting data

Doing a manual full sync on the browser extension fixes the issue.

**Comment by BlackDex** [comment, marked as answer]
This seems to be a known issue with the extensions and full sync should solve it as you already noticed.

See: https://github.com/bitwarden/browser/issues/1928
